io.github.bkbilly.lnxlink.yaml

```diff
@@ -0,0 +1,50 @@
+id: io.github.bkbilly.lnxlink
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: lnxlink
+modules:
+  - flatpak-modules-meson.yaml
+  - flatpak-modules-scikit.yaml
+  - flatpak-modules.yaml
+  - flatpak-modules-opencv.yaml
+  - flatpak-lnxlink.yaml
+  - flatpak-modules-system.json
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: file
+        path: io.github.bkbilly.lnxlink.metainfo.xml
+finish-args:
+  # Wayland access
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  # DBus
+  # - --socket=system-bus
+  # - --socket=session-bus
+  # For DBus
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.login1
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.mpris.MediaPlayer2.*
+  # Access to network
+  - --share=network
+  # Access to audio
+  - --socket=pulseaudio
+  # Access to Bluetooth
+  - --allow=bluetooth
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+  # Access to filesystem
+  - --filesystem=/tmp/
+  - --filesystem=/var/run/reboot-required
+  # host-os
+  - --filesystem=host-os
```

Comment:
why do you need this?

Reply:
It seems that it works without it, so I removed it.

Reply:
Don't hardcode `tmp` use the tempfile library https://docs.python.org/3/library/tempfile.html

Then set TMPDIR in the finish-args.